src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterSuggestTests.cs

```diff
@@ -170,5 +170,41 @@ public void TestSuggestVariableName(string suggestion)
             Assert.Equal(BindKind.PowerFxResolvedObject, nameInfo.Kind);
             Assert.IsType<NameSymbol>(nameInfo.Data);
         }
+
+        [Theory]
+        [InlineData("d", "displayName")]
+        [InlineData("D", "displayName")]
+        [InlineData("di", "displayName")]
+        [InlineData("DI", "displayName")]
+        [InlineData("dis", "displayName")]
+        [InlineData("DIs", "displayName")]
+        [InlineData("display", "displayName")]
+        [InlineData("displayname", "displayName")]
+        [InlineData("l", "logicalB")]
+        [InlineData("L", "logicalB")]
+        [InlineData("lo", "logicalB")]
+        [InlineData("LO", "logicalB")]
+        [InlineData("logical", "logicalB")]
+        [InlineData("logicalB", "logicalB")]
+        public void TestSuggestIdentifier(string txt, string expected)
```

Comment:
>TestSuggestIdentifier [](http://example.com/codeflow?start=20&length=21)

Can this be in Core  (not interpreter) 

See existing tests at: 
[https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs](https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs)

That also has a pattern that can simplify the test authoring (and avoid things like hardcoding the constant 21),